packaging/epel/convert2rhel.spec

```diff
@@ -11,37 +17,51 @@ License:        GPLv3+
 URL:            https://github.com/oamg/convert2rhel
 Source0:        %{url}/archive/v%{version}/%{name}-%{version}.tar.gz
 BuildArch:      noarch
-BuildRequires:  python-devel
-BuildRequires:  python-setuptools
+
 BuildRequires:  epel-rpm-macros
```

Comment:
@Conan-Kudo, is this dependency necessary? If I drop it, I could merge both spec files we have into one.

I know that it includes the license related macro, but I can define it in the spec myself (`%{!?_licensedir:%global license %%doc}`).

We have one that's used to build a package on CentOS/Oracle Linux, where `epel-rpm-macros` is not available, and then another spec, now almost the same, for building a package for EPEL.

Reply:
You shouldn't need it.